go.sum

```diff
@@ -1,6 +1,11 @@
 cloud.google.com/go v0.26.0/go.mod h1:aQUYkXzVsufM+DwF1aE+0xfcU+56JwCaLick0ClmMTw=
 github.com/BurntSushi/toml v0.3.1 h1:WXkYYl6Yr3qBf1K79EBnL4mak0OimBfB0XUf9Vl28OQ=
 github.com/BurntSushi/toml v0.3.1/go.mod h1:xHWCNGjB5oqiDr8zfno3MHue2Ht5sIBksp03qcyfWMU=
+github.com/CESARBR/knot-cloud-storage v0.0.0-20200311161130-348492da29a9 h1:wma+1Z0K4H/L1j+5ZXJ2c58xS0lW6GQKlCPK38Ab/AM=
```

Comment:
These `go.sum` changes shouldn't be in this commit. I suggest you include them only when a new package is added based on the commit needs. Otherwise, changes that can't be placed in a specific commit should be added into a new one.